desktop/ui/src/main/java/org/datacleaner/windows/AbstractDatastoreDialog.java

```diff
@@ -49,6 +52,27 @@
     private final JButton _saveButton;
     private final JButton _cancelButton;
     private final UserPreferences _userPreferences;
+    
+    protected final JXTextField _datastoreNameTextField;
```

Comment:
I will let that be up to you. I can live with both ways of doing it ("quick'n'dirty" or "more work") :-) Up to you!